components/auth/SelectPrincipal.vue

```diff
@@ -163,6 +181,10 @@ export default {
 </template>
 
 <style lang="scss" scoped>
+  .search-slot{
+    color: var(--body-text);
+  }
+
   .select-principal {
     &.retain-selection {
       min-height: 86px;
```

Comment:
Not related to this issue or changes here at all, but the reason for the bobble of content below this component on selection of a principal is caused by this value being too small, it should be `min-height: 91px;`.  Could you make this tweak whilst you're in the area?